pkgs/applications/audio/pavucontrol/default.nix

```diff
@@ -10,6 +10,14 @@ stdenv.mkDerivation rec {
     sha256 = "1qhlkl3g8d7h72xjskii3g1l7la2cavwp69909pzmbi2jyn5pi4g";
   };
 
+  patches = [
+    # Can be removed with the next version bump
```

Comment:
```suggestion
    # Can be removed with the 4.1 version bump
```

Reply:
I wouldn't bet on that, the last releases updated the major version. I think it is fine as is, since a package bump needs to touch the patches anyway, since it won't build otherwise.